presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -1015,6 +1015,18 @@ var QueryDetail = React.createClass({
                                         { query.session.source }
                                     </td>
                                 </tr>
+                                  {(() => {
```

Comment:
I don't think you need the `if`. Allowing the field to be null is fine, and more consistent with the rest of the UI.